<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>---</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Exp</string>
         </attribute>
         <widget class="QPushButton" name="btnConfirm">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>71</width>
            <height>71</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSliderRobot">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>200</y>
            <width>1641</width>
            <height>51</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>OpenHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>630</x>
            <y>630</y>
            <width>601</width>
            <height>281</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>10</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Total:</string>
           </property>
          </widget>
          <widget class="QLCDNumber" name="lcdNumberTotal">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>30</y>
             <width>131</width>
             <height>71</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>10</y>
             <width>121</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Round total:</string>
           </property>
          </widget>
          <widget class="QLCDNumber" name="lcdNumberRound">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>30</y>
             <width>131</width>
             <height>71</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="intValue" stdset="0">
            <number>10</number>
           </property>
          </widget>
          <widget class="QPushButton" name="btnStartExperiment">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>571</width>
             <height>111</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>START</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="geometry">
           <rect>
            <x>660</x>
            <y>40</y>
            <width>491</width>
            <height>121</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLCDNumber" name="lcdNumberPlayerInvestment">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>30</y>
             <width>131</width>
             <height>71</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>10</y>
             <width>141</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Robot received:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>10</y>
             <width>141</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Robot investment:</string>
           </property>
          </widget>
          <widget class="QLCDNumber" name="lcdNumberRobotInvestment">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>30</y>
             <width>131</width>
             <height>71</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_confirm_btn">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>490</y>
            <width>1411</width>
            <height>131</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QPushButton" name="pushButton_0">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_1">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_5">
           <property name="geometry">
            <rect>
             <x>650</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_6">
           <property name="geometry">
            <rect>
             <x>780</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_7">
           <property name="geometry">
            <rect>
             <x>900</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_8">
           <property name="geometry">
            <rect>
             <x>1020</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_9">
           <property name="geometry">
            <rect>
             <x>1140</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_10">
           <property name="geometry">
            <rect>
             <x>1260</x>
             <y>20</y>
             <width>99</width>
             <height>91</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Admin</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>220</y>
            <width>271</width>
            <height>201</height>
           </rect>
          </property>
          <property name="title">
           <string>Connection Box</string>
          </property>
          <widget class="QLineEdit" name="lineEditNaoIP">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>151</width>
             <height>27</height>
            </rect>
           </property>
           <property name="inputMask">
            <string>000.000.000.000</string>
           </property>
           <property name="text">
            <string>192.168.0.1</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>151</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>NAO IP Address:</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnConnectToNao">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>251</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>30</y>
             <width>61</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEditNaoPort">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>50</y>
             <width>81</width>
             <height>27</height>
            </rect>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>9559</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>90</y>
            <width>291</width>
            <height>121</height>
           </rect>
          </property>
          <property name="title">
           <string>XML file</string>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>151</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>XML path:</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEditXML">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>50</y>
             <width>211</width>
             <height>61</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="btnBrowse">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>50</y>
             <width>51</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>50</y>
            <width>391</width>
            <height>451</height>
           </rect>
          </property>
          <property name="title">
           <string>Robt Control</string>
          </property>
          <widget class="QPushButton" name="btnWakeUp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>361</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string>Init Position</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnRest">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>361</width>
             <height>81</height>
            </rect>
           </property>
           <property name="text">
            <string>Rest</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnFaceTrackingEnable">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>240</y>
             <width>361</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string>Enable Face Tracking</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnFaceTrackingDisable">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>340</y>
             <width>361</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string>Disable Face Tracking</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
